pkg/channel/consolidated/dispatcher/dispatcher.go

```diff
@@ -90,13 +89,15 @@ func (sub Subscription) String() string {
 
 func NewDispatcher(ctx context.Context, args *KafkaDispatcherArgs) (*KafkaDispatcher, error) {
 	conf := sarama.NewConfig()
-	conf.Version = sarama.V2_0_0_0
+	// don't get the clientID cleared by when it is built later
```

Comment:
I'm not sure I understand this comment

Reply:
Let me add more explanation here.
But, it is about an explicit setting of the clientId here in the caller of the `BuildSaramaConfig`.